docs/source/ecf/guidance.html.md

```diff
@@ -706,119 +706,100 @@ For more detailed information see the specifications for this [notify that an EC
 }
 ```
 
-### View data for all participants who have transferred
+###  Update a replacement mentor’s schedule
 
-<div class="govuk-inset-text">This endpoint is only available for systems integrated with API v3 onwards. They will not return data for API v1 or v2.</div>
+A new mentor can be assigned to an ECT part way through training, replacing the ECT’s original mentor.
 
-As soon as school induction tutors have entered the information to the DfE service, providers can view data for participants who have transferred:
+[Providers must notify the DfE of replacement mentors by updating their training schedule.](/api-reference/ecf/guidance.html#notify-dfe-of-a-participant-39-s-training-schedule)
 
-* **from** schools they are in partnership with
-* **to** schools they are in partnership with
+Note, if a replacement mentor is already mentoring another ECT and they replace a mentor for a second ECT, the first ECT takes precedence. In this instance, the provider should not change the mentor’s schedule.
 
-```
-GET /api/v3/participants/ecf/transfers
-```
+Providers must include a `schedule_identifier` reflecting when the replacement mentor starts.
+
+* `ecf-replacement-september`
+* `ecf-replacement-january`
+* `ecf-replacement-april`
+
+For API v3 onwards, a replacement mentor's schedule, and any associated declaration submissions, do not need to align with the ECT they are mentoring.
+
+Previously for API v1 and v2, a replacement mentor could start mentoring an ECT part way through their training. The provider had already submitted a `start` declaration for the previous mentor (in line with the ECT). If the provider were to submit a `retention-1` declaration for the ECT, then any declaration submitted for the new replacement mentor in the same milestone period would also have to be a retention-1 declaration. This is no longer the case for API v3.
+
+## Participant transfers (v3 API-only) 
```

Comment:
could we make sure that this is v3 onwards? so we don't have to go back and update if this is moved to RECT?